<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ant Detector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,3">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;黑体&quot;;
background-color: red;
</string>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="controlTab">
       <attribute name="title">
        <string>控制</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="cameraButton">
          <property name="text">
           <string>启动相机</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoDetectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>自动检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="manualDetectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>手动检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="systemSettingsButton">
          <property name="text">
           <string>系统设置</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>systemSettingsButton</zorder>
       <zorder>autoDetectButton</zorder>
       <zorder>manualDetectButton</zorder>
       <zorder>textBrowser</zorder>
       <zorder>cameraButton</zorder>
      </widget>
      <widget class="QWidget" name="standerImageTab">
       <attribute name="title">
        <string>标准图</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="standerImageListWidget"/>
        </item>
        <item>
         <widget class="QPushButton" name="addStanderImageButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>拍摄标准图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeStanderImageButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>删除标准图</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingTab">
       <attribute name="title">
        <string>检测设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="settingListWidget"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>笔刷粗度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addSettingButton">
          <property name="text">
           <string>添加检测设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="testImageTab">
       <attribute name="title">
        <string>测试图</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="correctLabel">
          <property name="text">
           <string>正确列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="correctImageListWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="correctImageButtonLayout">
          <item>
           <widget class="QPushButton" name="addCorrectImageButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>拍摄</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importCorrectImageButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cleanCorrectImageListButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="errorLabel">
          <property name="text">
           <string>错误列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="errorImageListWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="errorImageButtonLayout">
          <item>
           <widget class="QPushButton" name="addErrorImageButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>拍摄</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importErrorImageButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cleanErrorImageListButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="batchDetectButton">
          <property name="text">
           <string>批量检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeTestImageButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>删除测试图</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="cameraAction">
   <property name="text">
    <string>启动相机</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="shortcutVisibleInContextMenu" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="autoDetectAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>启动自动检测</string>
   </property>
  </action>
  <action name="manualDetectAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>手动检测</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>相机设置</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>自动检测设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
